parkingspaces/src/main/java/com/baspiotr/parkingspaces/components/TimeComponent.java

```diff
@@ -0,0 +1,15 @@
+package com.baspiotr.parkingspaces.components;
+
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
+@Component
```

Comment:
you should delete the class - it is redundant. you can use LocalDateTime.now() and this is not any sort of problem